### ISSUE TITLE
Media gallery doest not appear for image field

### ISSUE THREAD

**Issue by MrZyr0** [htmlburger/carbon-fields#1265]
## Version

  - Carbon Fields: 3.6.5
  - WordPress: 6.7.1
  - PHP: 8.1.30

## Expected Behavior

When I click on the button "Select Image" ![Screenshot 2024-11-22 at 11 00 18](https://github.com/user-attachments/assets/58fb6246-f497-4478-b66a-10b42b2beb96), the WordPress media gallery should appear

## Actual Behavior

Clicking does nothing

## Container definition
```php
Block::make('images-grid', __('Images grid', 'example-com'))
    ->set_description(__('Massonery pictures grid', 'example-com'))
    ->set_category('media')
    ->set_icon('grid-view')
    ->set_keywords([__('massonery', 'example-com'), __('media', 'example-com'), __('pictures', 'example-com'), __('photos', 'example-com'), __('gallery', 'example-com'), __('slider', 'example-com')])
    ->set_mode('both')
    ->add_fields([
      Field::make('text', 'total_columns', __('Grid width', 'example-com'))
        ->set_attribute('type', 'number')
        ->set_attribute('min', 1)
        ->set_attribute('max', 12)
        ->set_default_value(4)
        ->set_help_text(__('Define the number of grid columns.', 'example-com')),
      Field::make('complex', 'masonry_images', __('Images', 'example-com'))
        ->add_fields(array(
          Field::make('image', 'image', __('Image', 'example-com'))
            ->set_type('image')
            ->set_value_type('url'),
          Field::make('text', 'columns', __('Columns size', 'example-com'))
            ->set_attribute('type', 'number')
            ->set_attribute('min', 1)
            ->set_attribute('max', 12)
            ->set_default_value(1)
            ->set_help_text(__('Indicate the space taken up by the image in width.', 'example-com')),
          Field::make('text', 'rows', __('Lines size', 'example-com'))
            ->set_attribute('type', 'number')
            ->set_attribute('min', 1)
            ->set_default_value(1)
            ->set_help_text(__('Indicate the space taken up by the image in height.', 'example-com'))
        ))
        ->set_min(1)
        ->set_help_text(__('Add images to fill the grid.', 'example-com')),
    ])
```

## Steps to Reproduce the Problem

  1. Create new page
  2. Add the block
  3. Add an entry to the complex
  4. Try to select an image

## Comments

D'après mes investigations, il y aurait un problème de déclaration de l'évènement car j'ai plusieurs sites sur le même environnement (déclaration Docker similaire avec le même thème, la même version de WordPress et les mêmes plugins sous la même version) et je n'ai pas la même fonction rattachée.
Pour le site ayant un bouton fonctionnel j'ai `un()` qui a du code exécutable, mais pour le site qui ne fonctionne pas j'ai `noop()` qui est une fonction vide.

Here is some screenshots from the devtool of my non working website :

<img width="1624" alt="Screenshot 2024-11-22 at 11 10 27" src="https://github.com/user-attachments/assets/8c1d1a39-618c-41e0-bd64-5556b28ae18c">

<img width="1624" alt="Screenshot 2024-11-22 at 11 10 48" src="https://github.com/user-attachments/assets/c08fbd1f-c262-4f06-969b-80697f4606f6">

Here is some screenshots from the devtool of my working website :

<img width="1624" alt="Screenshot 2024-11-22 at 11 12 54" src="https://github.com/user-attachments/assets/fc800c41-52e8-4b71-a7a6-ccb8fb537fc6">

<img width="1624" alt="Screenshot 2024-11-22 at 11 13 05" src="https://github.com/user-attachments/assets/84198ce1-c80c-4a4d-b7e1-c21ee821d579">

### Actual Behavior

https://github.com/user-attachments/assets/39177101-3860-4957-bfd1-88cff42e0c1c

### Expected Behavior

https://github.com/user-attachments/assets/f4a9678e-a464-40b6-b0d0-e53d89642146

**Comment by MrZyr0** [comment]
I seem to have this problem on all fields that should open the media gallery.
Below is a video where I have the same problem with a `media_gallery` field.

https://github.com/user-attachments/assets/c7ee5598-52df-422a-a655-610246e049ac

**Comment by piterka** [comment]
I have the same problem... @MrZyr0 were you able to fix this somehow?

**Comment by MrZyr0** [comment]
No, for now the only bypass is to change the values in the database.

The block is displayed despite everything, it is really only the opening of the gallery that poses a problem.

**Comment by MrZyr0** [comment]
🆙 
Does anyone have any ideas to help me debug this problem?
I have nothing in debug.log, no errors in the console about this. Nothing at all to point me in the right direction.
What's more, I can't reproduce the problem in my local environment.
With the same `docker-compose`, the same database and the same files, I don't have the problem.

It only seems to exist in my pre-production environment.
What's surprising is that on this same environment, I have two websites built with the same theme (totally ISO, only the content differs) and I've only solved the problem on one of them by deactivating and reactivating my theme and the Yoast SEO plugin. It does nothing on the other site.

I'm totally stuck at this stage


Here's my plugins list :
- Carbon Fields v3.6.5 (installed with pre-autoloaded package)
- Redirection v5.5.1
- WP Mail SMTP v4.3.0
- WPML Media Translation v2.7.5
- WPML Multilingual CMS v4.6.15
- WPML SEO v2.1.0.1
- WPML String Translation v3.2.17
- Yoast SEO v24.0

WordPress v6.7.1

**Comment by lkuijf** [comment]
Setting SCRIPT_DEBUG to false will make the button work.
#1268

**Comment by MrZyr0** [comment]
It change nothing because I enable debugging like this :
```
define( 'WP_DEBUG_LOG', true );
define( 'SCRIPT_DEBUG', false );
define( 'WP_DEBUG', true );
define( 'WP_DEBUG_DISPLAY', true );
ini_set('display_errors', 1);
ini_set('display_startup_errors', 1);
error_reporting(E_ALL);
```

So the `SCRIPT_DEBUG` is never `true`

**Comment by Cheff-Jeff** [comment]
@MrZyr0  I have the same problem on version 3.6.9 were you able to fix this?

**Comment by MrZyr0** [comment]
> @MrZyr0  I have the same problem on version 3.6.9 were you able to fix this?

Not at all. I just not encountered this problem on production but I didn't know why.

**Comment by ciromattia** [comment]
I got the same issue on 6.8 and fixed setting SCRIPT_DEBUG to false.
